pkgs/development/tools/selenium/chromedriver/default.nix

```
@@ -46,9 +48,13 @@ in stdenv.mkDerivation rec {
     install -m755 -D chromedriver $out/bin/chromedriver
   '' + lib.optionalString (!stdenv.isDarwin) ''
     patchelf --set-interpreter ${glibc.out}/lib/ld-linux-x86-64.so.2 $out/bin/chromedriver
-    wrapProgram "$out/bin/chromedriver" --prefix LD_LIBRARY_PATH : "${libs}:\$LD_LIBRARY_PATH"
+    wrapProgram "$out/bin/chromedriver" --prefix LD_LIBRARY_PATH : "${libs}"
   '';
 
+  passthru.tests = {
+    version-check = callPackage ./tests.nix { };
+  };
+
```

```suggestion
  passthru.tests.version =
    testVersion { package = chromedriver; };

```

https://github.com/NixOS/nixpkgs/pull/121896

---

Thank you!